packages/grafana-toolkit/src/cli/tasks/plugin.utils.ts

```diff
@@ -131,28 +138,28 @@ const prepareRelease = useSpinner<any>('Preparing release', async ({ dryrun, ver
 
 interface GithubPluglishReleaseOptions {
```

Comment:
`Pluglish`, is this an internal joke of Plugins team? :)

Reply:
Did you laugh? Haha, I thought it was pretty funny :) But seriously folks

Reply:
I did! <3

Reply:
Fixed